src/routes/protected.tsx

```diff
@@ -290,15 +289,19 @@ export const protectedRoutes: RouteNode[] = [
           {
             path: ProjectDashboardRoutes.REFERRALS,
             element: (
-              <ProjectEditRoute
-                permissions={[
-                  'canManageIncomingReferrals',
-                  'canManageOutgoingReferrals',
-                ]}
-                redirectRoute={Routes.PROJECT}
-              >
-                <ProjectReferralsPage />
-              </ProjectEditRoute>
+              // No ProjectRoute wrapper due to complexity: some parts of page are visible for Legacy referrals,
+              // other parts visible if CE is enabled, etc. ProjectCeReferralsPage component handles permissions internally.
+              // See useProjectCeVisibility hook for details.
+              <ProjectCeReferralsPage />
```

Comment:
If this component now renders both CE and Legacy referrals, maybe to reduce confusion could we rename it `ProjectReferralsPage`?